app/controllers/rails_admin_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class RailsAdminController < ApplicationController
   # RailsAdmin does not include translations. Therefore, we fallback to English locales
-  skip_around_action :switch_locale
+  skip_before_action :set_locale
```

Comment:
Not sure if thats okay but the around_action did not work with the new set_locale code. Do you know if the around_action is required for something?

Reply:
The `around_action` is not strictly required, but I am wondering why it wasn't working? That shouldn't make much of a difference...

Reply:
I was trying the `around_action` once again, but couldn't find an issue: [around_action.patch](https://github.com/openHPI/codeharbor/files/14466846/around_action.patch)
<img width="766" alt="Bildschirmfoto 2024-03-02 um 00 21 27" src="https://github.com/openHPI/codeharbor/assets/7300329/27801771-3e7d-4883-ac4c-e3edc2f28efa">

Maybe I am missing some action, some email that is sent or some other integration?

Reply:
It works for me as well now. The problem I had before was that no page on CodeHarbor would load/display anything even though the console reported that the view had been rendered. Not sure what caused that but apparently it had something to do with my setup. I am applying your patch.